react-function-components/src/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function CustomButton() {
+  return <button type='button'>Click Me!</button>;
+}
+
+ReactDOM.render(
+  <CustomButton/>,
+  document.querySelector('#root')
+);
```

Comment:
This is the old way of doing this, you need to be using the new way for React 18.
```JSX
const container = document.querySelector('#root');
const root = ReactDOM.createRoot(container);

root.render(<CustomButton />);
```